### PR TITLE
Rename scale's `bandSize` to `rangeStep`

### PR DESCRIPTION
**Pull request by kanitw**
(very likely this will fail codecov test for diff coverage since it touches so many files)

### CODE REVIEW

**Comment by domoritz** [comment]
Upping your loc count, I see :-P

**Comment by kanitw** [comment]
> Upping your loc count, I see :-P

Hard to beat adding / removing interface like you lol. 

Honestly, this PR is a pretty boring task. If you had done it for me, I'd greatly appreciate. (Too late!)

File: src/scale.ts [hunk 1]
```diff
@@ -90,13 +90,13 @@ export interface ScaleConfig {
    *  Default band width for `x` ordinal scale when is mark is `text`.
```
Comment (domoritz) [inline src/scale.ts]: Are we changing the descriptions as well?

Comment (kanitw) [inline src/scale.ts]: Good catch, I'll search for all occurrences for band width and band size

**Comment by kanitw** [comment]
@domoritz Ok force pushed the fix for description

File: test/compile/mark/bar.test.ts [hunk 2]
```diff
@@ -236,7 +236,7 @@ describe('Mark: Bar', function() {
     });
     const props = bar.properties(model);
 
-    it('should use x and with with band size true and 0.5 offset', () => {
+    it('should use y and with band true', () => {
```
Comment (domoritz) [inline test/compile/mark/bar.test.ts]: "should use y with band true"

**Comment by domoritz** [comment]
LGTM after the one comment fix.

**Pull request merged**